smpte.js

```diff
@@ -422,6 +422,55 @@ function insertBibliography(docMetadata) {
   h2.innerText = "Bibliography";
 }
 
+const SMPTE_ELEMENTS_ID = "sec-elements";
+
+function insertElementsAnnex(docMetadata) {
+  const sec = document.getElementById(SMPTE_ELEMENTS_ID);
+
+  if (sec === null)
+    return;
+
+  if (sec.children.length !== 1 || sec.firstElementChild.tagName !== "OL") {
+    logEvent(`Elements section must contain a single <ol> element.`);
+    return;
+  }
+
+  sec.classList.add("annex");
+
+  const intro = document.createElement("p");
+  intro.innerText = "This annex lists non-prose elements of this document."
+  sec.insertBefore(intro, sec.firstChild);
+
+  const h2 = document.createElement("h2");
+  h2.innerText = "Non-prose elements";
```

Comment:
Still wrong header section name

```suggestion
  h2.innerText = "Additional elements";
```